Cargo.toml

```diff
@@ -32,7 +32,7 @@ python = ["pyo3"]
 
 [dependencies]
 abe_policy = "1.0"
-cosmian_crypto_core = "2.0"
+cosmian_crypto_core = { git = "https://github.com/Cosmian/crypto_core.git", branch = "develop" }
```

Comment:
Use `CryptoCore` 3.0?

Reply:
The problem is that I depend on additions post-3.0 that are on develop. I could use the v3.0 if I get rid of the KDF stuff.

Reply:
Is it possible to release a 3.1 before this PR?

Reply:
No since it is on `develop` and tags are only for `main`.

Reply:
But we could merge develop into main and tag a 3.1 if the changes on develop are complete and non breaking?

Reply:
@tbrezot understandable, but please at least pin it to a specific commit using `rev = "38fb7c7"`

Reply:
done

Reply:
done